2025/ideas-list.md

```diff
@@ -6,6 +6,7 @@ page of each organization under the NumFOCUS umbrella at this page.
 
 - [aeon](https://github.com/aeon-toolkit/aeon-admin/blob/main/gsoc/gsoc-2025-projects.md)
 - [ArviZ](https://github.com/arviz-devs/arviz/wiki/GsoC-2025-projects)
+- [AiiDA](https://github.com/aiidateam/aiida-core/wiki/GSoC-2025-Projects)
```

Comment:
Thanks @unkcpz, can you keep it in sorted order please.

Reply:
Sure, updated.